package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "react-spatial",
   "description": "Components to build React map apps.",
-  "version": "1.0.13",
+  "version": "1.0.14-beta.4",
   "license": "MIT",
   "dependencies": {
     "@geops/geops-ui": "0.1.8",
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.56",
```

Comment:
Do we want to use material-ui in react-spatial?

Reply:
Since we use it in all our app it makes sense to have the default component using MUI. @mhaertwig ?
But it's important to have the possibility to use something else if needed.

Reply:
we could also use the MUI autocomplete only in the example.